examples/ipython/langevin_integrator_check.ipynb

```diff
@@ -122,7 +122,7 @@
    "outputs": [],
    "source": [
     "for i in range(nframes):\n",
-    "    # generate the next frame (which is sim.nsteps_per_iteration timesteps)\n",
+    "    # generate the next frame (which is sim.n_steps_per_iteration timesteps)\n",
```

Comment:
`n_steps_per_iteration` => `n_steps_per_frame` (in a comment so it doesn't really matter, but let's go ahead and fix that now)